src/main/scala/tech/cryptonomic/conseil/tezos/michelson/parser/JsonParser.scala

```diff
@@ -88,6 +88,16 @@ object JsonParser {
     override def toMichelsonExpression = MichelsonStringConstant(string)
   }
 
+  /*
+   * Wrapper for bytes constant
+   *
+   * {"bytes": "0500"}
+   *
+   * */
+  case class JsonBytesConstant(bytes: String) extends JsonExpression {
```

Comment:
Would it be useful to wrap the `bytes` String in a Value class and only parse formally correct byte values with a custom parser/decoder, as we do with JsonDecoders for TezosTypes?

Reply:
I was thinking about it but decided not to do it because it would require guessing validation rules and since I have only a few examples, the guessing might not match the reality.